core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -137,6 +137,7 @@
     public static final String ENABLE_LARGE_DYNAMIC_FILTERS = "enable_large_dynamic_filters";
```

Comment:
this doesn't fix issue mentioned in https://github.com/trinodb/trino/pull/15154. That query doesn't have top-n.